programs/sll_remove_dups.py

```diff
@@ -0,0 +1,26 @@
+from single_linked_lists import Node
+
+
+def remove_duplicates(head: Node) -> None:
+    p1 = head  # p1 is ponter 1
+    while p1.next:
```

Comment:
What happens with an empty list?

Reply:
If the list is empty or the list has only one element then in remove_duplicates the while p1.next doesn´t happed
and this function does nothing